src/components/FormValidator.js

```diff
@@ -78,7 +78,7 @@ export default class FormValidator {
     this._submitButton = this._formElement.querySelector(
```

Comment:
**Could be improved**

You could make a special method `resetValidation` for clearing errors and controlling the submit button.
It could look something like this:

```js

resetValidation() {
      this._toggleButtonState(); <== toggle the submit button ==

      this._inputList.forEach((inputElement) => {
        this._hideError(inputElement) <== clear errors ==
      });

    }



```

And you can call it in `index.js` when you click on the profile open button.